tests/airgapped/README.md

```diff
@@ -0,0 +1,91 @@
+# Testing Airgapped Installation
+
+For running the tests we expect an environment that can:
+1. Spin up LXC containers
+2. Have Docker, to pull images
+
+We need docker to pull all the images necessary and push them to the airgapped
+lxc container.
+
+## Setup
+
+We've prepared some scripts for setting up the environment
+```bash
+./tests/airgapped/setup/setup.sh
```

Comment:
`pigz` is also needed for some of the commands that you will run. Please add a note or a command in the setup.sh

Reply:
This is needed to run the `scripts/airgapped/*` files, so I'll include this in the installation script there, which gets called by the above `setup.sh` script
https://github.com/canonical/bundle-kubeflow/pull/682/commits/b154e4afce1c52ff2732879f59477763cf879fe3